### ISSUE TITLE
how to run CLI when installed using docker on Ubuntu

### ISSUE THREAD

**Issue by holmeslice92** [iron-fish/ironfish#179]
Hi there, 

Looking for help on how to run ironfish after installing using docker on ubuntu. I just get "command not found" when using ironfish start in the command line. 

If I try run the docker command again (the one under the install instruction for linux) it says that the another node is using the database. 

Complete noob so any help in layman's terms would be greatly appreciated. 

Thanks,

**Comment by dguenther** [comment]
If another Docker container is using the database, you can run `docker ps` and then run `docker kill` on any running nodes in other Docker containers to stop them.

Unfortunately we're not yet especially easy to install and set up for layman users, but if you're willing to join [our Discord](https://discord.gg/EkQkEcm8DH), there are plenty of users willing to help other users get their nodes up and running. Aside from that, we're definitely planning to make nodes easier to download/install in the future.
